hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/repository/RecordFileRepository.java

```diff
@@ -30,4 +31,11 @@ public interface RecordFileRepository extends PagingAndSortingRepository<RecordF
 
     @Query("select max(r.index) from RecordFile r")
     Optional<Long> findLatestIndex();
+
+    @Cacheable(value = "record_file", key = "record_index")
```

Comment:
This is putting all cache entries under the same key. By default it will use all args as cache key.
```suggestion
    @Cacheable(cacheNames = "record_file.index", cacheManager = EvmConfiguration.CACHE_MANAGER_10M, unless = "#result == null")
```